.github/workflows/create_llvm.yml

```diff
@@ -88,10 +97,17 @@ jobs:
         run:
           sudo apt-get install --yes g++-11-aarch64-linux-gnu
 
+      - name: install x86 build tools
+        if: ${{ steps.cache.outputs.cache-hit != 'true' && matrix.arch == 'x86' }}
+        run: |
+          sudo dpkg --add-architecture i386
+          sudo apt-get update
+          sudo apt-get install --yes gcc-multilib g++-multilib libc6-dev:i386 lib32tinfo-dev
```

Comment:
Your other change for setup_ubuntu installs this. So we may want to combine in some way at some point.